### ISSUE TITLE
推理耗时问题：deim_dinov3_s模型测试平均耗时46ms，但是同样的图片使用yolo11s推理耗时平均14~15ms

### ISSUE THREAD

**Issue by deepxiaobai** [Intellindust-AI-Lab/DEIMv2#121]
我的测试代码是：
deim_dinov3_s
import colorsys
import torch
from torchvision import transforms
from PIL import Image, ImageDraw, ImageFont
from safetensors.torch import load_file
import torch.nn as nn
from huggingface_hub import PyTorchModelHubMixin

from engine.backbone import HGNetv2, DINOv3STAs
from engine.deim import HybridEncoder, LiteEncoder
from engine.deim import DFINETransformer, DEIMTransformer
from engine.deim.postprocessor import PostProcessor

import time

class DEIMv2(nn.Module, PyTorchModelHubMixin):
    def __init__(self, config):
        super().__init__()
        if 'DINOv3STAs' in config:
            self.backbone = DINOv3STAs(**config["DINOv3STAs"])
        else:
            self.backbone = HGNetv2(**config["HGNetv2"])
        if 'LiteEncoder' in config:
            self.encoder = LiteEncoder(**config["LiteEncoder"])
        else:
            self.encoder = HybridEncoder(**config["HybridEncoder"])
        if 'DEIMTransformer' in config:
            self.decoder = DEIMTransformer(**config["DEIMTransformer"])
        else:
            self.decoder = DFINETransformer(**config["DFINETransformer"])
        self.postprocessor = PostProcessor(**config["PostProcessor"])

    def forward(self, x, orig_target_sizes):
        t1 = time.time()

        x = self.backbone(x)
        t2 = time.time()
        print(f"backbone {t2 - t1}")
        x = self.encoder(x)
        t3 = time.time()
        print(f"encoder {t3 - t2}")
        x = self.decoder(x)
        t4 = time.time()
        print(f"decoder {t4 - t3}")

        x = self.postprocessor(x, orig_target_sizes)
        print(f"postprocessor {time.time() - t4}")

        return x

deimv2_s_config = {
  "DINOv3STAs": {
    "name": "vit_tiny",
    "embed_dim": 192,
    "interaction_indexes": [
      3,
      7,
      11
    ],
    "num_heads": 3
  },
  "HybridEncoder": {
    "in_channels": [
      192,
      192,
      192
    ],
    "feat_strides": [
      8,
      16,
      32
    ],
    "hidden_dim": 192,
    "use_encoder_idx": [
      2
    ],
    "num_encoder_layers": 1,
    "nhead": 8,
    "dim_feedforward": 512,
    "dropout": 0.0,
    "enc_act": "gelu",
    "expansion": 0.34,
    "depth_mult": 0.67,
    "act": "silu",
    "version": "deim",
    "csp_type": "csp2",
    "fuse_op": "sum"
  },
  "DEIMTransformer": {
    "feat_channels": [
      192,
      192,
      192
    ],
    "feat_strides": [
      8,
      16,
      32
    ],
    "hidden_dim": 192,
    "num_levels": 3,
    "num_layers": 4,
    "eval_idx": -1,
    "num_queries": 300,
    "num_denoising": 100,
    "label_noise_ratio": 0.5,
    "box_noise_scale": 1.0,
    "reg_max": 32,
    "reg_scale": 4,
    "layer_scale": 1,
    "num_points": [
      3,
      6,
      3
    ],
    "cross_attn_method": "default",
    "query_select_method": "default",
    "activation": "silu",
    "mlp_act": "silu",
    "dim_feedforward": 512,
    "eval_spatial_size": [640, 640],
  },
  "PostProcessor": {
    "num_top_queries": 300
  },
}
device = torch.device("cuda:0")
deimv2_s = DEIMv2(deimv2_s_config)
# deimv2_s_hf = DEIMv2.from_pretrained("Intellindust/DEIMv2_DINOv3_S_COCO")

local_safetensors_path = "./weights/DEIMv2_DINOv3_S_COCO_model.safetensors"
state_dict = load_file(local_safetensors_path)

# 步骤3：加载权重到模型（同上，处理前缀）
state_dict = {k.replace("module.", ""): v for k, v in state_dict.items()}
deimv2_s.load_state_dict(state_dict, strict=False)
deimv2_s.to(device)
deimv2_s.eval()

print("本地safetensors权重加载完成！")

MODEL = deimv2_s
IMAGE_SIZE = (640, 640)  # resolution settings of model, you can find in config
IMAGE_PATH = "example.jpg"  # path of input image
CONFIDENCE_THRESHOLD = 0.4

label_map = {
    1: 'person', 2: 'bicycle', 3: 'car', 4: 'motorcycle', 5: 'airplane',
    6: 'bus', 7: 'train', 8: 'truck', 9: 'boat', 10: 'traffic light',
    11: 'fire hydrant', 12: 'stop sign', 13: 'parking meter', 14: 'bench',
    15: 'bird', 16: 'cat', 17: 'dog', 18: 'horse', 19: 'sheep',
    20: 'cow', 21: 'elephant', 22: 'bear', 23: 'zebra', 24: 'giraffe',
    25: 'backpack', 26: 'umbrella', 27: 'handbag', 28: 'tie', 29: 'suitcase',
    30: 'frisbee', 31: 'skis', 32: 'snowboard', 33: 'sports ball', 34: 'kite',
    35: 'baseball bat', 36: 'baseball glove', 37: 'skateboard', 38: 'surfboard', 39: 'tennis racket',
    40: 'bottle', 41: 'wine glass', 42: 'cup', 43: 'fork', 44: 'knife',
    45: 'spoon', 46: 'bowl', 47: 'banana', 48: 'apple', 49: 'sandwich',
    50: 'orange', 51: 'broccoli', 52: 'carrot', 53: 'hot dog', 54: 'pizza',
    55: 'donut', 56: 'cake', 57: 'chair', 58: 'couch', 59: 'potted plant',
    60: 'bed', 61: 'dining table', 62: 'toilet', 63: 'tv', 64: 'laptop',
    65: 'mouse', 66: 'remote', 67: 'keyboard', 68: 'cell phone', 69: 'microwave',
    70: 'oven', 71: 'toaster', 72: 'sink', 73: 'refrigerator', 74: 'book',
    75: 'clock', 76: 'vase', 77: 'scissors', 78: 'teddy bear', 79: 'hair drier',
    80: 'toothbrush'
}

num_classes = len(label_map)
category_colors = {}
for i, label in enumerate(label_map.values()):
    hue = i / num_classes
    saturation = 0.8
    value = 0.9
    rgb = colorsys.hsv_to_rgb(hue, saturation, value)
    rgb = tuple(int(c * 255) for c in rgb)
    category_colors[label] = rgb

transform = transforms.Compose([
    transforms.Resize(IMAGE_SIZE),
    transforms.ToTensor(),
])
image = Image.open(IMAGE_PATH).convert("RGB")
input_tensor = transform(image).unsqueeze(0).to(device)
orig_target_sizes=torch.tensor([IMAGE_SIZE]).to(device)
t1 = time.time()
for i in range(100):
    if i==1:
        t1 = time.time()
    # MODEL.eval()
    with torch.no_grad():
        outputs = MODEL(input_tensor, orig_target_sizes=orig_target_sizes)
print((time.time() - t1)/99)

在A100上测试输出如下：

<img width="630" height="686" alt="Image" src="https://github.com/user-attachments/assets/2e6a2f8b-3466-40a1-80da-98c09c7bdf03" />

我应该如何调整，才能获取和报告中一样，比yolo11s推理耗时更少的结果
